### ISSUE TITLE
Restic backup performance is very low

### ISSUE THREAD

**Issue by beer4duke** [beer4all/rclone#11]
Current performance is really low when using the rclone xrootd backend:
![image](https://user-images.githubusercontent.com/6848916/84380194-3f899e00-abe7-11ea-83e4-415ec7f98eff.png)

Or also:
![image](https://user-images.githubusercontent.com/6848916/84380253-562ff500-abe7-11ea-871b-e82e51084eeb.png)

This could be linked to checksum computations in rclone?

It would be interesting to see what we obtain with different rclone settings, like `--size-only --ignore-checksum` or increase the number of parallel transfers
See [here](https://gitlab.cern.ch/restic/rclone/-/wikis/Rclone-performance)

Having a look at restic cache options could also be interesting...

**Comment by kiderrien** [comment]
It seems to work fine on small files:
```
[userxroot@resticrclone3 ~]$ restic -r rclone:xrootd:tmp/backup backup /home/userxroot/rdev/
enter password for repository:
repository d753f8cd opened successfully, password is correct

Files:        3226 new,     0 changed,     0 unmodified
Dirs:            2 new,     0 changed,     0 unmodified
Added to the repo: 239.425 MiB

processed 3226 files, 240.945 MiB in 0:07
snapshot 3a821ed8 saved
```
but it has trouble for files larger than 500 MiB. (see [here](https://gitlab.cern.ch/restic/rclone/-/wikis/Restic-Rclone-XrootD#result-1) )

I did some tests with rclone serve restic, which allows to use rclone restic as a server (see [here](https://rclone.org/commands/rclone_serve_restic/) ) to be able to try with the flags `--size-only --ignore-checksum`  but the tests are catastrophic

first server:
`rclone serve restic -v xrootd:tmp/backup_killian --size-only --ignore-checksum`
```
[userxroot@resticrclone3 ~]$ restic backup /home/userxroot/rdev
repository 4fbeef7f opened successfully, password is correct
created new cache in /home/userxroot/.cache/restic
[8:48] 46.13%  189 files 111.153 MiB, total 3226 files 240.945 MiB, 0 errors ETA 10:16

Files:        3226 new,     0 changed,     0 unmodified
Dirs:            2 new,     0 changed,     0 unmodified
Added to the repo: 240.782 MiB

processed 3226 files, 240.945 MiB in 20:54
```
the overall time is to be reviewed because it was uninterrupted is it resumed where it was in was. The overall time is to be reviewed because it was uninterrupted is it resumed where it was. I think it was between 30 and 40 min 


second server:
`rclone serve restic -v xrootd:tmp/backup_killian --size-only --ignore-checksum`
```
[userxroot@resticrclone3 ~]$ restic backup /home/userxroot/rdev
repository ac6e857f opened successfully, password is correct
created new cache in /home/userxroot/.cache/restic
[19:18] 58.96%  594 files 142.050 MiB, total 3226 files 240.945 MiB, 0 errors ETA 13:26

Files:        3226 new,     0 changed,     0 unmodified
Dirs:            2 new,     0 changed,     0 unmodified
Added to the repo: 240.782 MiB

processed 3226 files, 240.945 MiB in 31:08
snapshot 3798f8b9 saved
```

we are on time of several ten minutes with rclone serve restic against seconds for restic server clone.

**Comment by kiderrien** [comment]
rclone serve restic seems quite effective with ssh. it just needs a few seconds for the rdev file compared to xrootd where currently it takes tens of minutes.

### SSH ###
#### First server: ####
`rclone serve restic -v ssh:backup `
```
[userxroot@resticrclone3 ~]$ restic backup /home/userxroot/rdev/
repository d67f3826 opened successfully, password is correct
created new cache in /home/userxroot/.cache/restic

Files:        3226 new,     0 changed,     0 unmodified
Dirs:            2 new,     0 changed,     0 unmodified
Added to the repo: 240.169 MiB

processed 3226 files, 240.945 MiB in 0:04
snapshot 2d0677cb saved
```
#### Second server: ####
`rclone serve restic -v ssh:backup --size-only --ignore-checksum`

```
[userxroot@resticrclone3 ~]$ restic backup /home/userxroot/rdev/
repository e5a24e96 opened successfully, password is correct
created new cache in /home/userxroot/.cache/restic

Files:        3226 new,     0 changed,     0 unmodified
Dirs:            2 new,     0 changed,     0 unmodified
Added to the repo: 240.044 MiB

processed 3226 files, 240.945 MiB in 0:04
```
#### Third server: #### 
`rclone serve restic -v xrootd:tmp/backup_killian --size-only --ignore-checksum --transfers 20`
```
[userxroot@resticrclone3 ~]$ restic backup /home/userxroot/rdev/
repository 99f1246c opened successfully, password is correct
created new cache in /home/userxroot/.cache/restic

Files:        3226 new,     0 changed,     0 unmodified
Dirs:            2 new,     0 changed,     0 unmodified
Added to the repo: 238.585 MiB

processed 3226 files, 240.945 MiB in 0:05
```

**Comment by beer4duke** [comment]
This is now fixed with 67d02950a performance is now nominal.
